README.md

```diff
@@ -70,7 +70,7 @@ When a new release (or version) occurs, change the Unreleased section of the [CH
 
 See also the list of [contributors](../../graphs/contributors) who participated in this project.
 
-## License
+## license
```

Comment:
Leave uppercase

Reply:
Thank you for your feedback. I have changed 'license' back to uppercase.

Reply:
Thank you!